roles/kubernetes-apps/container_engine_accelerator/nvidia_gpu/defaults/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+container_engine_accelerator_enabled: false
```

Comment:
I think this variable should be renamed to something like “nvidia_accelerator_enabled”. NVIDIA GPUs are not the only type of accelerator (they’re not even the only kinds of GPUs).

Reply:
i agree and will also be in next commit.